apps/site/components/EOL/EOLModal/index.tsx

```diff
@@ -67,13 +44,14 @@ const EOLModal: FC<ModalProps<EOLModalData>> = ({
           </p>
         )}
 
-        <VulnerabilitiesTable vulnerabilities={knownVulnerabilities} />
-
-        <UnknownSeveritySection
-          vulnerabilities={unknownVulnerabilities}
-          open={knownVulnerabilities.length > 0}
+        <VulnerabilitiesTable
```

Comment:
@ovflowd I'm not sure if you'd prefer a new `KnownVulnerabilitySection` component which wraps this component + the filtering, as you mentioned encapsulating this logic.

Reply:
Go for it.